testHelpers/utils.ts

```diff
@@ -79,14 +86,18 @@ class Utils {
   }
 
   setContracts(
-    erc1155721: ERC1155ERC721,
+    //erc1155721: ERC1155ERC721,
+    voucherSets: VoucherSets,
+    vouchers: Vouchers,
     voucherKernel: VoucherKernel,
     cashier: Cashier,
     bsnRouter: BosonRouter,
     bsnTokenPrice?: MockERC20Permit,
     bsnTokenDeposit?: MockERC20Permit
   ): void {
-    this.contractERC1155ERC721 = erc1155721;
+    //this.contractERC1155ERC721 = erc1155721;
```

Comment:
please remove this line

Reply:
Removed. Will be included in next commit.